tests/integration/test_iam.py

```diff
@@ -16,6 +16,7 @@ class TestIAMIntegrations(unittest.TestCase):
     def setUp(self):
         self.iam_client = aws_stack.create_external_boto_client("iam")
 
+    # TODO what does this test do?
```

Comment:
nit: Have you found out what it does? Does the TODO have to stay in the code? :D

Reply:
It will stay for now, but might be removed in the future :D